api/src/test/java/org/pmiops/workbench/utils/TestMockFactory.java

```diff
@@ -42,16 +46,22 @@ public Workspace createWorkspace(String workspaceNameSpace, String workspaceName
     ResearchOutcomeEnumsList.add(ResearchOutcomeEnum.IMPROVED_RISK_ASSESMENT);
 
     return new Workspace()
-        .id("1")
+        .id(WORKSPACE_FIRECLOUD_NAME)
```

Comment:
There's still a bit of a fib happening here, because the workspace model passed in to createWorkspace has fewer fields set than the model used by other methods. I'll make this clearer in the next PR.